packages/inspector-extension/src/index.ts

```diff
@@ -50,7 +50,7 @@ const inspector: JupyterFrontEndPlugin<IInspector> = {
   ): IInspector => {
     const { commands, shell } = app;
     const command = CommandIDs.open;
-    const label = 'Open Contextual Help';
+    const label = 'Contextual Help';
```

Comment:
Since `title` and `label` are the same, can we consolidate them to just `label` now?

Reply:
good call! fixed that now.